Subject: [PATCH] Add postinit to skip ALSA default configuration

The default codec values are different between amixer & kernel codec driver.
The reason is that alsactl_init sets strange default configurations during booting.
We add postinit to skip this. The default values are set by kernel codec.

Index: alsa-utils-1.0.27.2/alsactl/init/Makefile.am
===================================================================
--- alsa-utils-1.0.27.2.orig/alsactl/init/Makefile.am
+++ alsa-utils-1.0.27.2/alsactl/init/Makefile.am
@@ -1,7 +1,7 @@
 
 init_files = \
 	00main default help info test \
-	hda ca0106
+	hda ca0106 postinit
 EXTRA_DIST = $(init_files)
 alsainitdir = $(datadir)/alsa/init
 alsainit_DATA = $(init_files)
Index: alsa-utils-1.0.27.2/alsactl/init/postinit
===================================================================
--- /dev/null
+++ alsa-utils-1.0.27.2/alsactl/init/postinit
@@ -0,0 +1,2 @@
+# [Advantech] Do not do ALSA default configuration
+RESULT="true", EXIT="return"
